src/components/Header.scss

```diff
@@ -0,0 +1,55 @@
+header {
+  display: flex;
```

Comment:
- [ ] You have used `display: flex` in this file 9 times. This is a bad practice because you are repeating yourself a lot. I would suggest you create a class which has only one job to make the display flex and you give to classes accordingly